docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -41,7 +186,7 @@ A basic read-only connector should implement the following methods:
 
 If you are interested in seeing strategies for implementing more methods,
 look at the :doc:`example-http` and the Cassandra connector. If your underlying
-data source supports schemas, tables and columns, this interface should be
+data source supports schemas, tables, and columns, this interface should be
```

Comment:
Not sure if this is better. Also unrelated to current PR, so let's skip it here

Reply:
Oh, I ran the Vale docs linter on the file and didn't notice it's for an existing paragraph. Would you mind if I addressed the findings in a separate commit? The Vale linter uses the Google developer documentation guide, which we strive to follow (it's in the README in this module). This is an example of an Oxford comma.

Reply:
Okay, let's address it in a separate commit.
Is this linter something we would run as part of CI as well to catch any deviations from guideline ? (similar to running error prone or check style on the Java source code)

Reply:
Yes, when we address all the errors. That's why I'm starting to slowly do this in regular PRs. I'll open a bigger PR for other docs later.